librubyfmt/src/parser_state.rs

```diff
@@ -158,8 +157,7 @@ impl ParserState {
     }
 
     pub fn insert_comment_collection(&mut self, comments: CommentBlock) {
-        self.comments_to_insert
-            .merge(comments.apply_spaces(self.spaces_after_last_newline));
+        self.comments_to_insert += comments.apply_spaces(self.spaces_after_last_newline);
```

Comment:
I am not sure if I hate this or if I love this. I will think about it

Reply:
It was just the most convenient way to get it onto `Option` but we can do a trait with a named method